hw/bsp/imxrt/boards/mimxrt1050_evkb/board.h

```diff
@@ -45,7 +45,7 @@
 
 // UART
 #define UART_PORT             LPUART1
-#define UART_RX_PINMUX        IOMUXC_GPIO_AD_B0_13_LPUART1_RX
-#define UART_TX_PINMUX        IOMUXC_GPIO_AD_B0_12_LPUART1_TX
+#define UART_RX_PINMUX        IOMUXC_GPIO_AD_B0_13_LPUART1_RXD
+#define UART_TX_PINMUX        IOMUXC_GPIO_AD_B0_12_LPUART1_TXD
```

Comment:
@robert-hh rt1050 only has issue with the symbol naming in later mcu-sdk, fixed already.